Library/Homebrew/extend/pathname.rb

```diff
@@ -343,14 +343,19 @@ def ensure_writable
     chmod saved_perms if saved_perms
   end
 
+  # @private
+  def find_install_info
```

Comment:
```suggestion
  def install_info
```
Will need renaming below too.

Reply:
Ah, you're right. Sorry; was not paying attention!